Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/AzureWebAppMvpModel.java

```diff
@@ -696,6 +699,18 @@ public void clearWebAppsCache() {
         subscriptionIdToWebApps.clear();
     }
 
+    public List<Region> getAvailableRegions(String subscriptionId, PricingTier pricingTier) throws IOException {
```

Comment:
handle null pricingTier

Reply:
pricingTier is not required in the rest api, here is the [reference](https://docs.microsoft.com/en-us/rest/api/appservice/listgeoregions/listgeoregions#uri-parameters)

Reply:
if pricingTier is null, NPE will be thrown: pricingTier.toSkuDescription().tier()